### PR TITLE
[UPG] Fetchmail inbox invoice.

### PR DESCRIPTION
**Pull request by StefanRijnhart**
Leaving out the integration with fetchmail_invoice. It can be implemented in
a dedicated glue module.

### CODE REVIEW

File: fetchmail_inbox_invoice/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+# Copyright (C) 2014 Therp BV (<http://therp.nl>)
```
Comment (NL66278) [inline fetchmail_inbox_invoice/__openerp__.py:2]: Add your own copyright with 2017 after this

File: fetchmail_inbox_invoice/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Fetch mail into inbox (invoice specific)",
+    "version": "8.0.1.0",
```
Comment (NL66278) [inline fetchmail_inbox_invoice/__openerp__.py]: 8.0.1.0.0 according to OCA standards

File: fetchmail_inbox_invoice/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "author": "Therp BV",
+    "complexity": "normal",
+    "description": """
```
Comment (NL66278) [inline fetchmail_inbox_invoice/__openerp__.py]: put description in README

File: fetchmail_inbox_invoice/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    "depends": [
+        'account',
+        'fetchmail_inbox',
+    ],
+    "data": [
+        'view/mail_message.xml',
+        'view/menu.xml',
+        'security/ir.model.access.csv',
+    ],
+    "js": [
```
Comment (NL66278) [inline fetchmail_inbox_invoice/__openerp__.py]: Leave out empty keys

**Comment by NL66278** [comment]
@StefanRijnhart Thanks for migration! Just put your own copyright in all files after original copyright in all files with significant changes.

File: fetchmail_inbox_invoice/__openerp__.py (hunk 4 above)
Comment (StefanRijnhart) [reply fetchmail_inbox_invoice/__openerp__.py]: done

File: fetchmail_inbox_invoice/__openerp__.py (hunk 3 above)
Comment (StefanRijnhart) [reply fetchmail_inbox_invoice/__openerp__.py]: done

File: fetchmail_inbox_invoice/__openerp__.py (hunk 2 above)
Comment (StefanRijnhart) [reply fetchmail_inbox_invoice/__openerp__.py]: done

File: fetchmail_inbox_invoice/__openerp__.py (hunk 1 above)
Comment (StefanRijnhart) [reply fetchmail_inbox_invoice/__openerp__.py:2]: Thanks for the review! I don't think this deserves copyright, but I accepted a mention under contributors in the README.

**Pull request merged**